kpops/component_handlers/kubernetes/utils.py

```diff
@@ -26,3 +29,27 @@ def trim(max_len: int, name: str, suffix: str) -> str:
         )
         return new_name
     return name
+
+
+def validate_image_tag(image_tag: str) -> str:
```

Comment:
imo the returned str is a bit useless here since it is never changed from the original input. I think raising the exception if it's invalid is enough and aligns better with the name of the function